core/common/src/PlanarClipMask.ts

```diff
@@ -47,71 +42,85 @@ export enum PlanarClipMaskPriority {
   /** A reality model with a bounded range. */
   RealityModel = 0,
   /** A design model stored in the [IModelDb]($backend). */
-  BIM = 2048,
+  DesignModel = 2048,
   /** The maximum supported priority. */
   Maximum = 4096,
```

Comment:
@RBBentley is this a hard limit for some technical reason, such that PlanarClipMaskSettings should be enforcing it?
Is there correspondingly a minimum priority?

Reply:
There really isn't a limit -- probably could remove this.     The relative magnitude only is used.   In general these only time you would want to use anything but defaults is if you wanted to order the relative masking order of a set of reality models.